### PR TITLE
comps updates for 2.8

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: comps/comps-katello-pulp-server-rhel7.xml [hunk 1]
```diff
@@ -67,8 +67,6 @@
        <packagereq type="default">python-qpid</packagereq>
        <packagereq type="default">python-qpid-common</packagereq>
        <packagereq type="default">python-qpid-qmf</packagereq>
-       <packagereq type="default">python-requests</packagereq>
-       <packagereq type="default">python-rhsm</packagereq>
```
Comment (ehelms) [inline comps/comps-katello-pulp-server-rhel7.xml:71]: We still need python-rhsm for rhel6?

Comment (jlsherrill) [inline comps/comps-katello-pulp-server-rhel7.xml:71]: we do, as its not in centos6

**Comment by ehelms** [comment]
ACK

File: comps/comps-katello-pulp-server-rhel7.xml [hunk 2]
```diff
@@ -67,9 +68,6 @@
...
-       <packagereq type="default">python-semantic-version</packagereq>
```
Comment (jlsherrill) [inline comps/comps-katello-pulp-server-rhel7.xml:72]: and this comes from epel on el7

**Comment by ehelms** [comment]
ACK

**Pull request merged**